src/Compilers/Core/Portable/Symbols/IPropertySymbol.cs

```diff
@@ -76,6 +76,11 @@ public interface IPropertySymbol : ISymbol
         /// </summary>
         IMethodSymbol? SetMethod { get; }
 
+        /// <summary>
+        /// The backing field of the property, or null if the property does not use a backing field.
```

Comment:
Think the doc comment could be a bit more descriptive here. As written I think it could be interpreted as both user and compiler defined backing fields while this should be only supporting the former.

Reply:
How does this sound?
```suggestion
        /// The compiler-generated backing field of the property, or null if one is not generated.
```